tsconfig.base.json

```diff
@@ -27,47 +27,122 @@
     "pretty": true,
     "preserveWatchOutput": true,
     "suppressImplicitAnyIndexErrors": true,
+    "ignoreDeprecations": "5.0",
```

Comment:
I think you should fix deprecation instead :D

Reply:
as discussed, https://github.com/AmadeusITGroup/otter/issues/439 has been created